fluentui_others/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerView.kt

```diff
@@ -292,6 +293,14 @@ class PeoplePickerView : TemplateView {
         peoplePickerTextViewAdapter?.showSearchDirectoryButton = showSearchDirectoryButton
 
         updateHintVisibility()
+        if (context.isAccessibilityEnabled) {
```

Comment:
is this required ?

Reply:
Yes, we do not want to have keyboard focus on these when not in talkback mode as we do not have any action on these.
But in talkback, these should be announced